pkg/cmd/browse/browse.go

```diff
@@ -131,15 +133,19 @@ func runBrowse(opts *BrowseOptions) error {
 	if err != nil {
 		return err
 	}
-	url := ghrepo.GenerateRepoURL(baseRepo, section)
+	// taken from cli/internal/ghrepo/repo.go
+	url := fmt.Sprintf("%s%s/%s", ghinstance.HostPrefix(baseRepo.RepoHost()), baseRepo.RepoOwner(), baseRepo.RepoName())
+	if section != "" {
+		url = fmt.Sprintf("%s/%s", url, section)
+	}
```

Comment:
Also, just to be clear, I changed this because the encoded characters act like format specifiers for the `fmt.Sprintf(p, args...)` in `GenerateRepoURL()`. I copied the code as a simple hack around it?

Reply:
You can keep using GenerateRepoURL, but like this so to avoid issues with `%`:
```
GenerateRepoURL(baseRepo, "%s", section)
```
Note that `section` should already be path-escaped here